RussianFilter/sections/adservers.txt

```diff
@@ -231,6 +231,7 @@
 ||prfctmney.com^$third-party
 ||knwekg.com^$third-party
 ||vftbnl.com^$third-party
+||spyglass.reklamko.pro^$third-party
```

Comment:
I don't see this request.

Reply:
![image](https://user-images.githubusercontent.com/68852737/93756837-67d55f80-fc0e-11ea-94d1-fd48f7f0f455.png)